src/data/faq.json

```diff
@@ -87,6 +87,117 @@
                     }
                 ]
             },
+            {
+                "title": "Schnelltest",

```

Comment:
```suggestion
                "title": "Quick Tests",

```

Reply:
ok, but I think through CWA it is call rapid test or rapid antigen test.

Reply:
Then let's use rapid test.

Reply:
@dsarkar Modified the suggestion.

Reply:
@dsarkar Modified the suggestion.

Reply:
thanks